profiles/views.py

```diff
@@ -26,11 +27,17 @@ def profile_detail(request, github_account, template_name="profiles/profile.html
 
     extra_fields = ExtraField.objects.filter(profile=profile)
 
-    return render(
-        request,
-        template_name,
-        {"local_profile": profile, "user": profile.user, "extra_fields": extra_fields},
-    )
+    context = {
+        "local_profile": profile,
+        "user": profile.user,
+        "extra_fields": extra_fields,
+    }
+    if profile.share_favorites:
+        context["favorite_packages"] = Package.objects.filter(
+            favorite__favorited_by=profile.user
+        )
```

Comment:
Let's limit this to the top 25 packages. We can add pagination later. 

```suggestion
        context["favorite_packages"] = Package.objects.filter(
            favorite__favorited_by=profile.user
        )[:25]
```

Reply:
I'll do that! Thanks Jeff!